From e461c3b0935a05a169aa22a1b113c08f8989eea6 Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@users.noreply.github.com>
Date: Sun, 9 Jun 2024 15:09:36 -0600
Subject: [PATCH 1/1] Fix documentation

contributing.rst refers to a non-existant 'bug-fixes' doc

conf.py refers to non-existant '_static' directory

Signed-off-by: hololeap <hololeap@users.noreply.github.com>
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -49,7 +49,7 @@ html_theme = 'sphinx_rtd_theme'
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
 
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
2.39.2

